src/tooltip.js

```diff
@@ -6,13 +6,16 @@
  * @example
  *
  *<button
- *        ts-tooltip-event="click"                              // Options are click or mouseover
  *        ts-tooltip="Something that shows up in there"         // The text to show in the tooltip
  *        ts-tooltip-direction="bottom"                         // The direction the tooltip pops up
+ *        ts-tooltip-event="click"                              // Options are click or mouseover
  *        ts-tooltip-show="someModel.someBoolean"               // A boolean if set will use this instead of events
+ *        ts-tooltip-id="String"                                // A string used to identify the tooltip
  *        ts-tooltip-class="someClass"                          // Adds class to tooltip container
  *        ts-tooltip-content-hover="someModel.someBoolean"      // A boolean if set will allow users to
- *                                                              // hover over the tooltip content without closing it
+ *        ts-tooltip-show-close="someModel.someBoolean"         // A boolean to show close button
+ *        ts-tooltip-close-callback="someFunction"              // Event Handler when closed is clicked
+ *        ts-tooltip-margin="Integer"                           // Integer value to provide margin between tooltip and element
```

Comment:
Might be good to make this a string so whoever uses it can use units other than `px`.